### ISSUE TITLE
Fargate

### ISSUE THREAD

**Issue by himberjack** [silinternational/ecs-deploy#134]
Doesn't seem to work with Fargate-flagged tasks

**Comment by bramswenson** [comment]
```
An error occurred (InvalidParameterException) when calling the UpdateService operation: Task definition does not support launch_type FARGATE.
```

**Comment by bazaglia** [comment]
PR #129 should be merged to fix this.

**Comment by hguillermo** [comment]
Let's close this issue. It is working on the current version. #129 fixed it. @fillup @himberjack

**Comment by kakakakakku** [comment]
@fillup 
I contributed supporting Fargate via https://github.com/silinternational/ecs-deploy/pull/129.
So, we can close it 👍

**Comment by fillup** [comment]
Fixed in #129
